tasmota/tasmota_xx2c_global/xx2c_interface.ino

```diff
@@ -406,7 +406,11 @@ const uint8_t kI2cList[] = {
 #endif
 
 #ifdef XI2C_96
-  XI2C_96
+  XI2C_96,
+#endif
+
+#ifdef XI2C_97
+  XI2C_97
 #endif
```

Comment:
You cannot go beyond 96 as it breaks command I2CDriver which only supports up to 96.